### ISSUE TITLE
Respecting complex numbers

### ISSUE THREAD

**Issue by fb87** [Keno/SIUnits.jl#106]
Would be great if complex power could be named by `VA`, while real power stays `W` and imaginary power gets `Var`.

Depends on: #104 and #76.

**Comment by tlycken** [comment]
> As far as I understand Julia, complex numbers aren't separated in real and imaginary parts

That's incorrect (see [the source code](https://github.com/JuliaLang/julia/blob/387e964353559361587618aac557ad1ee6276f23/base/complex.jl#L3-L6)). However, Julia's complex numbers are typed as `Complex{T<:Real}` and since unitful quantities don't subtype `Real` they can't be used independently for the built-in variables.

Instead, we usually do `SIQuantity{Complex{T}, ...}`, in which case we don't separate units over the real and imaginary parts.

**Comment by fb87** [comment]
Not sure if I understand your answer correct, but am I right, that at least a switch from `W`to `VA` is possible when complex numbers are used in conjunction with SIUnits?
